crates/oxc_transformer/src/lib.rs

```diff
@@ -67,7 +67,7 @@ impl<'a> Transformer<'a> {
         allocator: &'a Allocator,
         source_type: SourceType,
         semantic: Semantic<'a>,
-        options: TransformOptions,
+        options: &TransformOptions,
```

Comment:
It feels like the Transfomer struct should own `TransformOptions`, since it is going to be constructed only once for normal usages.